readme/apps/screen_reader_accessibility.md

```diff
@@ -30,7 +30,7 @@ Screen readers usually support jumping between these different regions. Here's t
 
 ### Finding notes, tags, and notebooks
 
-It can sometimes be difficult to find a tag/note/notebook using Joplin's navigation sidebars. In these cases, the ["Go to Anything" dialog](./search.md#goto-anything) can simplify accessing an item.
```

Comment:
I think with this we'd lose the normal Markdown linking? The documentation is setup so that it can be read directly on GitHub, while I think here the link would only work on the website?

That's why most links are in the form `https://github.com/laurent22/joplin/blob/dev/readme/apps/sync/index.md` in the doc, and they are converted when building the website.

Or maybe we could still have `./search.md` but modify the build script so that it handles these links too